video_worker/utils.py

```diff
@@ -1,7 +1,14 @@
 """
 Common utils.
 """
+import os
 import urllib
+import yaml
+
+
+DEFAULT_CONFIG_FILE_NAME = 'instance_config.yaml'
```

Comment:
this should ideally be `worker_config.yaml`, but its alright if it goes out of scope of this PR.